src/main/java/com/xml/RegularUtil.java

```diff
@@ -0,0 +1,66 @@
+package com.xml;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class RegularUtil {
+    private static final String PHONE_NUMBER = "^\\+\\d{2}\\(\\d{3}\\)\\d{3}\\-\\d{2}\\-\\d{2}$";
```

Comment:
Naming is not very good. It is not a phone number. It is phone number regular expression. 
`PHONE_NUMBER -> PHONE_NUMBER_REQULAR_EXPRESSION`
The same for the rest constants.